content/src/service/synchronization/streaming/StreamHelper.ts

```diff
@@ -28,7 +28,7 @@ export function streamMap<T1, T2>(mapping: (value: T1) => T2) {
   })
 }
 
-export function mergeStreams(readables: Readable[]): NodeJS.ReadableStream {
+export function mergeStreams(readables: Readable[]): Readable {
```

Comment:
I updated this, becase ReadableStream doesn't have the method destroy

And checked here `https://github.com/grncdr/merge-stream/blob/master/index.js` that the library returns a Passthrough wich is also a Readable